src/libcore/fmt/mod.rs

```diff
@@ -298,7 +298,7 @@ impl<'a> Display for Arguments<'a> {
 ///
 /// For more information on formatters, see [the module-level documentation][module].
```

Comment:
Yeah, `libstd` is mostly reexports of other, possibly unstable crates (collections, core, alloc).